src/common.js

```diff
@@ -206,6 +218,34 @@ exports.extend = extend;
 
 // Common wrapper for all Unix-like commands
 function wrap(cmd, fn, options) {
+  // -1 means "don't glob"
+  // 0 means "glob the options" (probably don't want this)
+  var indexOfExpandableArg = {
```

Comment:
Yeah, I was thinking about that approach as well. It's probably cleaner, so I'll modify it to do that instead. I'll get rid of the `-1` stuff too.